test/test-listing-guni.spec.ts

```diff
@@ -0,0 +1,233 @@
+import path from 'path';
+import { expect } from 'chai';
+import { config } from 'dotenv';
+
+import rawBRE, { ethers } from 'hardhat';
+
+import { BigNumber } from 'ethers';
+import { parseEther } from 'ethers/lib/utils';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address';
+import { JsonRpcSigner } from '@ethersproject/providers';
+import {
+  evmSnapshot,
+  increaseTime,
+  evmRevert,
+  latestBlock,
+  advanceBlockTo,
+  impersonateAccountsHardhat,
+  MAX_UINT_AMOUNT,
+} from './utils/utils';
+import { parsePoolData } from './utils/listing';
+import { IAaveGovernanceV2 } from '../types/IAaveGovernanceV2';
+import { IAaveOracle } from '../types/IAaveOracle';
+import { ILendingPool } from '../types/ILendingPool';
+import { AssetListingGUni } from '../types/AssetListingGUni';
+import { SelfdestructTransferFactory } from '../types/SelfdestructTransferFactory'
+import { IERC20 } from '../types/IERC20';
+
+const AAVE_GOVERNANCE_V2 = "0xEC568fffba86c094cf06b22134B23074DFE2252c"
+const AAVE_LENDING_POOL = "0x7937D4799803FbBe595ed57278Bc4cA21f3bFfCB" //'0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9';
+const VOTING_DURATION = 19200;
+
+const GUNI1 = "0x50379f632ca68D36E50cfBC8F78fe16bd1499d1e"
+
+const AAVE_WHALE = '0x25f2226b597e8f9514b3f68f00f494cf4f286491';
+const AAVE_TOKEN = '0x7fc66500c84a76ad7e9c93437bfc5ac33e2ddae9';
+
+const GUNI_HOLDER = '0x88215a2794ddC031439C72922EC8983bDE831c78';
+const AAVE_ORACLE = '0xA50ba011c48153De246E5192C8f9258A2ba79Ca9';
+
+const DAI_TOKEN = '0x6b175474e89094c44da98b954eedeac495271d0f';
+const DAI_HOLDER = '0x72aabd13090af25dbb804f84de6280c697ed1150';
+
+const ERRORS = {
+  NO_BORROW: '7',
+  NO_COLLATERAL_BALANCE: '9',
+  NO_STABLE_BORROW: '12',
+};
+
```

Comment:
need to load the `.guni.env` to make it work

Reply:
added above (L27-L36)